docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,18 @@
 
 ## September 2023
 
+**27 September 2023**
+
+Frontend version: 1.0.0-20230921.1438 | Backend version: 1.69.0-20230922.1123
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Storing API key for sending webhook to APIs | We are introducing the ability to store API Keys for sending webhooks to target APIs. With this feature, tenants will be able subscribe to and receive webhook events. |
+| **Feature** | Notifying tenants of expiring SAML app certificates | This new feature will notify tenants of expiring SAML app certificates. We want to provide tenants with more visibility on these expiring certificates so tenants can perform proper certificate rotation and ensure uninterrupted service. |
```

Comment:
We don't notify tenants, the notification is on the team's Slack channel, so it is only internal. This is informational for the team and don't need to appear in the release notes.

Reply:
oo ok! it wasnt marked w a * so i assumed tenants get notified.